### PR TITLE
Feature  basic layout

### PR DESCRIPTION
**Pull request by Ricecoder**

### CODE REVIEW

**Comment by cmauban** [comment]
Good job on getting two columns! Now work on those margins! :+1:

**Comment by nickycadavillo** [comment]
Great start! I agree with the margins. :+1: :octocat:

File: src/scss/_custom.scss [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+  clear: right;
+  display: inline-block;
+  width: 30%;
+  margin: 10px;
+}
+
+.rightcol {
+  background-color: green;
+  float: left;
+  display: inline-block;
```
Comment (al-the-x) [inline src/scss/_custom.scss:13]: Honestly, the `display: inline-block` property and `float: left` property are somewhat redundant. _Either_ float the columns or use `inline-block`, although the better choice here is `float`, I think.

File: src/scss/_custom.scss [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+  width: 60%;
+  margin: 10px;
```
Comment (al-the-x) [inline src/scss/_custom.scss:15]: Similarly, a better choice to get the two columns to play nice with one another is to use `padding` inside them and `box-sizing: border-box` as in the "Don't Overthink It Grids" article.

File: src/index.html [hunk 3]
```diff
@@ -20,7 +20,73 @@
...
+            <h3>1.1k</h3>
+            <h3>2</h3>
+            <h3>6</h3>
+          </span>
+
+          <span class="category">
+            <h5>Followers</h5>
+            <h5>Starred</h5>
+            <h5>Following</h5>
+          </span>
```
Comment (al-the-x) [inline src/index.html:49]: I think this text actually belongs with the `.numbers` above... If you check out the Reference Implementation, _both_ the numbers _and_ the words underneath are a part of the same link... Maybe you could put them together and make them `<big>`...?

**Comment by al-the-x** [comment]
I gave you some feedback on your profile markup, then I realized that this PR was _just_ for the layout elements...! Whoops!

**Pull request merged**